cmd/context/options.go

```diff
@@ -95,10 +95,11 @@ func (o *Options) InitFromEnvVars() {
 		usedEnvVars = append(usedEnvVars, model.OktetoNamespaceEnvVar)
 	}
 
-	if len(usedEnvVars) == 1 {
-		oktetoLog.Warning("Initializing context with the value of %s environment variable", usedEnvVars[0])
-	} else if len(usedEnvVars) > 1 {
-		oktetoLog.Warning("Initializing context with the value of %s and %s environment variables", strings.Join(usedEnvVars[0:len(usedEnvVars)-1], ", "), usedEnvVars[len(usedEnvVars)-1])
+	if o.Show {
```

Comment:
Why are we just showing this warning when the `o.Show` is set? I think this could prevent the user to know which context is using if the command is not a context one

Reply:
@jLopezbarb without this, if I run `okteto context show`, the warning appears breaking the JSON or YAML formatting. Do you have an alternative?

Reply:
good point @ifbyol, I've also noticed that `okteto endpoints --output=json` is affected by this. Will make a change before opening the PR for review again. Thanks both!